### DISCUSSION TITLE
ADNI data size after adni-to-bids

### DISCUSSION THREAD

**Discussion by mselimata** [aramis-lab/clinica#904] [Q&A] [answered]
Hi,
I noticed that once I run adni-to-bids the resulting BIDS formatted dataset is almost ten times smaller than the original ADNI dataset that I downloaded, Is that expected? 
Thanks.

**Comment by MatthieuJoulot** [comment, marked as answer]
Hi,
Yes that is absolutely normal ! Not all of the data is converted. We specifically choose some data. You can find some more details here: https://aramislab.paris.inria.fr/clinica/docs/public/latest/Converters/ADNI2BIDS/
